thorvg/examples/thorvg-example/main/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+idf_component_register(
+    SRCS "display_main.c"
+    INCLUDE_DIRS "."
+    PRIV_REQUIRES esp_lcd esp_timer esp_driver_ppa)
+
+idf_component_get_property(board_lib espressif__esp32_p4_function_ev_board_noglib COMPONENT_LIB)
```

Comment:
@espzav will you fix the build warning?

Reply:
What warning is there? I will try it on the board tomorrow.

Reply:
> What warning is there? I will try it on the board tomorrow.

@espzav , this warning.
```
espressif__esp32_p4_function_ev_board_noglib/esp32_p4_function_ev_board.c:33:31: warning: 'tp' defined but not used [-Wunused-variable]
   33 | static esp_lcd_touch_handle_t tp;   // LCD touch handle
```